benchmarks/benchmarks/common.py

```diff
@@ -0,0 +1,42 @@
+from functools import lru_cache
+from pathlib import Path
+import types
+import random
+
+import networkx as nx
+import nx_parallel as nxp
+
```

Comment:
I have added `__all__` and modified `__init__.py`, but I don't think it would be a good idea to add `timing_func`, `algo_types`, etc. in the main nx_parallel as they probably won't be used outside the `benchmarks`. The `timing_func` is a standard timing function that we will probably be using for most of the algorithms. I added it so that we wouldn't have to write the same thing again and again. Please let me know your thoughts on this.